framework_tvb/tvb/core/neocom/h5.py

```diff
@@ -223,8 +225,19 @@ def store_view_model(view_model, base_dir):
                     store_view_model(model_attr[idx], base_dir)
             else:
                 store_view_model(model_attr, base_dir)
+
+        if not references and isinstance(view_model, ViewModel) and not file_exist_before_save:
+            operation = dao.get_operation_by_id(os.path.basename(base_dir))
+            disk_size = get_view_model_disk_size(h5_path)
+            operation.disk_size = operation.disk_size + disk_size
+            dao.store_entity(operation)
+
```

Comment:
I would keep disk_usage computation and update in DB somewhere outside this method because we might need to use it from places that do not have access to DB.

One idea would be: after `h5.store_view_model()` is called from a service, we have on disk the operation folder with only view models inside. We could call at this point a method to compute the disk_size on the entire operation folder and then store it in DB?